src/features/DisplayControls/Treatments/components/Treatments.jsx

```diff
@@ -0,0 +1,161 @@
+import React, { useEffect } from "react";
+import {
+  DataTable,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableBody,
+  Table,
+  TableRow,
+  Link,
+  DataTableSkeleton,
+} from "carbon-components-react";
+import { FormattedMessage } from "react-intl";
+import { useState } from "react";
+import PropTypes from "prop-types";
+import { getPrescribedAndActiveDrugOrders } from "../utils/TreatmentsUtils";
+import "../styles/Treatments.scss";
+
+const Treatments = (props) => {
+  const { patientId } = props;
+  const [treatments, setTreatments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const treatmentHeaders = [
+    {
+      header: "Start Date",
+      key: "startDate",
+      isSortable: true,
+    },
+    {
+      header: "Drug Name",
+      key: "drugName",
+      isSortable: false,
+    },
+    {
+      header: "Dosage Details",
+      key: "dosageDetails",
+      isSortable: false,
+    },
+    {
+      header: "Prescribed By",
+      key: "prescribedBy",
+      isSortable: true,
+    },
+    {
+      header: "Actions",
+      key: "actions",
+      isSortable: false,
+    },
+  ];
+
+  const NoTreatmentsMessage = (
+    <FormattedMessage
+      id={"NO_TREATMENTS_MESSAGE"}
+      defaultMessage={"No IPD Medication is prescribed for this patient yet"}
+    />
+  );
+
+  const AddToDrugChart = (
+    <FormattedMessage
+      id={"ADD_TO_DRUG_CHART"}
+      defaultMessage={"Add to Drug Chart"}
+    />
+  );
+
+  const isIPDDrugOrder = (drugOrder) => {
+    return drugOrder.careSetting === "INPATIENT";
+  };
+
+  const setDosingInstructions = (drugOrder) => {
+    return (
+      drugOrder.dosingInstructions.dose +
+      " " +
+      drugOrder.dosingInstructions.doseUnits +
+      " - " +
+      drugOrder.dosingInstructions.route +
+      " - " +
+      drugOrder.dosingInstructions.frequency +
+      " - for " +
+      drugOrder.duration +
+      " " +
+      drugOrder.durationUnits
+    );
+  };
+
+  const modifyTreatmentData = (drugOrders) => {
+    const treatments = drugOrders.visitDrugOrders
+      .filter((drugOrder) => isIPDDrugOrder(drugOrder))
+      .map((drugOrder) => {
+        return {
+          id: drugOrder.uuid,
+          startDate: new Date(
+            drugOrder.effectiveStartDate
+          ).toLocaleDateString(),
```

Comment:
what is the date format? I think we are following DD MMM YYYY, please check and if that's the case create a common utility method in utils for formatting date and time and use it in here.